MapboxCoreNavigation/LocationManagers/NavigationLocationManager.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+import CoreLocation
+
+/**
+ `NavigationViewController` is the base location manager which handles
+ permissions and background modes.
+ */
+@objc(MBNavigationLocationManager)
+open class NavigationLocationManager: CLLocationManager {
+    
+    var lastKnownLocation: CLLocation?
+    
+    override public init() {
+        super.init()
+        
+        if CLLocationManager.authorizationStatus() == .notDetermined {
+            requestWhenInUseAuthorization()
+        }
+        
+        if Bundle.main.backgroundModeLocationSupported {
+            activityType = .automotiveNavigation
```

Comment:
Should the activity type always be `automotiveNavigation`, even when the user is getting walking directions?